recipes-bsp/u-boot/u-boot-imx_2021.04.bb

```diff
@@ -38,4 +38,4 @@ do_deploy:append:mx8m-nxp-bsp() {
 }
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
-COMPATIBLE_MACHINE = "(mx6-generic-bsp|mx7-generic-bsp|mx8-generic-bsp)"
+COMPATIBLE_MACHINE = "(imx-generic-bsp)"
```

Comment:
Not the same as it isn't expected to work with i.MX28 for example.

Reply:
Oh, I thought the `imx` override was only 6, 7, and 8. Was it? Is this a change with the new SOC overrides?

Reply:
No; also NXP produces the other i.MX so we ought to be consistent.